modules/charon-core/src/main/java/org/wso2/charon3/core/protocol/endpoints/UserResourceManager.java

```diff
@@ -661,6 +666,7 @@ public SCIMResponse updateWithPATCH(String existingId, String scimObjectString,
 
             User newUser = null;
 
+            Map<String, String> syncedAttributes = userManager.getSyncedUserAttributes();
```

Comment:
We will have to do a minor release for this neh? Since we are depending on new method in userManager which might not be implemented if some one uses a different user manager.

Reply:
I have added a default implementation for it https://github.com/wso2/charon/pull/422/files#diff-a58301cad2cf6876653fa20e6139d956a35ccaed67bc591fb0e8a2d4c7584620R307. Do we still need a minor release?

Reply:
Didn't see that. Were good.